scripts/tehn/earthsea.lua

```diff
@@ -24,6 +24,8 @@ local ripple_decay_rate = 1 / 0.5 / screen_framerate
 local ripple_growth_rate = 1 / 0.02 / screen_framerate
 local screen_notes = {}
 
+local MAX_NUM_VOICES = 16
```

Comment:
Is this something we want to define on the lua side or should it be an engine property?

Reply:
see discussion above... agree it can be cleaner but that is really a separate issue. without more work, lua and engine need to have independent but synchronized voice count

Reply:
> agree it can be cleaner but that is really a separate issue

Agreed, shall we create an issue for this?

Reply:
kinda part of https://github.com/monome/dust/issues/166